src/compile/common.ts

```diff
@@ -127,19 +127,19 @@ export function formatSignalRef(
 
 /**
  * Returns number format for a fieldDef
- *
- * @param format explicitly specified format
  */
 export function numberFormat(fieldDef: FieldDef<string>, specifiedFormat: string, config: Config) {
-  if (fieldDef.type === QUANTITATIVE) {
-    // add number format for quantitative type only
+  if (isTimeFieldDef(fieldDef)) {
+    return undefined;
+  }
 
-    // Specified format in axis/legend has higher precedence than fieldDef.format
-    if (specifiedFormat) {
-      return specifiedFormat;
-    }
+  // Specified format in axis/legend has higher precedence than fieldDef.format
```

Comment:
Isnt the comment outdated?  There is fieldDef.format anymore?

Reply:
Then it is

Reply:
We have ti for text: https://vega.github.io/vega-lite/docs/format.html